libraries/types/transaction/src/transaction.cpp

```diff
@@ -34,12 +35,26 @@ Transaction::Transaction(const trx_nonce_t &nonce, const val_t &value, const val
   getSender();
 }
 
+Transaction::Transaction(const bytes &_bytes, bool verify_strict, const h256 &hash) {
+  dev::RLP rlp;
+  try {
+    rlp = dev::RLP(_bytes);
+  } catch (const dev::RLPException &e) {
+    std::string error_msg =
```

Comment:
Do we have issue for that? if so maybe we should add TODO here, so it is easier to find what needs to be removed/changed